test/integration/config.json

```diff
@@ -16,10 +16,14 @@
     "pubsub": {
         "gcp_project_id": "test-project-id"
     },
+    "kinesis": {
+        "override_host": "http://kinesis:4567",
+        "max_retries" : 3
+    },
     "monitoring": {
         "prometheus_metrics_port": 9090,
         "profiler_port": 4269,
-        "profiling_path": "/tmp/tesla/tesla-telemetry/profile"
+        "profiling_path": "/tmp/tesla/fleet-telemetry/profile"
```

Comment:
nit: should this be `/tmp/fleet-telemetry/profile`?

Reply:
updated